index.md

```diff
@@ -13,9 +13,14 @@ Fabric is a lightweight, experimental modding toolchain for Minecraft.
 
 To install Fabric, you need the **Fabric Loader**, which you can <a class="page-link page-link-download" href="/use">download here</a>
 
-In addition, most mods depend on an API mod, containing commonly used hooks. They can be found here:
+You should also read the [Fabric Wiki installation instructions](https://fabricmc.net/wiki/install)!
 
-* [Fabric API for Minecraft 1.14 and above](https://minecraft.curseforge.com/projects/fabric/files)
+In addition, most mods depend on the Fabric API mod, containing commonly used hooks. It must be added to your mods folder. Download Fabric API for Minecraft 1.14 and above from here:
+
+* [CurseForge download for Fabric API](https://minecraft.curseforge.com/projects/fabric/files)
+* [Modrinth download for Fabric API](https://modrinth.com/mod/fabric-api/versions)
```

Comment:
Id prefer to keep just the CurseForge download link here.

Reply:
Why do you prefer to only have a CurseForge link? Is it the primary download? 

Still looks pretty clean and I believe that having a link to Modrinth will reduce support requests. Modrinth's featured versions are separate for every major version, such as 1.19.x, which would reduce support requests about having the wrong version. A lot of users come to Fabric for mods like Sodium or Iris, which are now only available on Modrinth, and would reduce support requests.

Reply:
A few reasons:

- I have some major concerns about Modrith's security, only the other day I asked if they had a Bug Bounty program and found out they didnt. I decided to have a quick poke around and in 10 mins (no joke) I had found a critical security flaw that effecively allowed me to remove any (even ones I dont own) from the site. I reported this and it was fixed quickly, but I believe the whole site needs a lot of work to improve things in this regard.
- The new CurseForge website makes it a lot easier to find the correct version.
- (Id be lying if I didnt mention this) Fabric isnt free to run, the revenue I made from CurseForge currently covers these costs, Modrith doesnt even come anywhere close to covering the running costs.

By linking to it we basically give it a seal of approval, right now in its current state I am not happy enough with it to do this. I currently have no plans to remove Fabric API from there.

Reply:
I see. I understand and respect that. Feel free to make those changes!! :)